### PR TITLE
search from multiple events indexes

### PR DESCRIPTION
**Pull request by ctdk**
This commit is very tiny, and should be entirely backwards compatible.

### CODE REVIEW

**Comment by Dieterbe** [comment]
Could we list the names of the indexes based on the time range we need instead of all indexes? Worth it?

**Comment by ctdk** [comment]
I imagine so - you can specify the indexes to search in a comma separated list. I hadn't done it that way because I didn't want to get ahead of myself and wasn't sure how to test it. I can leave a VM up gathering data for a few days though to test this out though.

**Comment by ctdk** [comment]
@Dieterbe: This PR now only searches the desired date range of events indexes, rather than wildcarding. Unfortunately this does break backwards compatibility with leaving the old events index around.

**Comment by Dieterbe** [comment]
hm right. good point. so the right thing is probably search in all indexes for now, and then in a few months update it to only look into the ones by date.

File: pkg/services/elasticstore/events.go [hunk 1]
```diff
@@ -42,7 +47,33 @@ func GetEventsQuery(query *m.GetEventsQuery) error {
...
+			for {
+				end = end.Add(-(time.Hour * 24))
+				y2, m2, d2 := end.Date()
+				idxDates = append(idxDates, fmt.Sprintf("events-%d-%02d-%02d", y, m, d))
+				if y2 <= y && m2 <= m && d2 <= d {
+					break
+				}
+			}
+		}
+		allTogether := strings.Join(idxDates, ",")
```
Comment (woodsaj) [inline pkg/services/elasticstore/events.go]: When testing this, i just get an empty string resulting in all indexes, including the "metric" index being searched.

**Pull request merged**